__tests__/cli-integration.test.js

```diff
@@ -2,17 +2,31 @@
 
 const { system, filesystem } = require('gluegun')
 const { expect } = require('chai')
+const fs = require('fs')
+const moveFolder = require('../utils/move-folder')
 
+const projectName = 'herbs-lab'
+const projectTmpName = 'project-tmp'
 const src = filesystem.path(__dirname, '..')
+const binPath = filesystem.path(src, 'bin', 'herbs')
+const tmpPath = filesystem.path(src, 'tmp', 'node_modules')
+const nodeModulesProjectTmpPath = filesystem.path(src, projectTmpName, 'node_modules')
+const projectGeneratedPath = filesystem.path(src, projectName)
 
-const cli = async (cmd) =>
-  system.run('node ' + filesystem.path(src, 'bin', 'herbs') + ` ${cmd}`)
-
-const projectName = 'herbs-lab'
+const cli = async (cmd) => system.run(`node ${binPath} ${cmd}`)
 
 describe('generates package.json', () => {
-  afterEach(() => {
-    filesystem.remove(projectName)
+  before(async () => {
```

Comment:
I think it need to be `beforeEach` instead before

Reply:
The before is used because it generates node_modules, so if you use beforeEach it will generate node_modules on every test